ci/run-tests.ps1

```diff
@@ -56,7 +57,7 @@ if ($run_with_spatial) {
         "-unattended", # Disable anything requiring user feedback
         "-nullRHI", # Hard to find documentation for, but seems to indicate that we want something akin to a headless (i.e. no UI / windowing) editor
         "-run=GenerateSnapshot", # Run the commandlet
-        "-MapPaths=`"$test_repo_map`"" # Which maps to run the commandlet for
+        "-MapPaths=`"$test_repo_map.Substring($test_repo_map.LastIndexOf("/")+1)`"" # Which maps to run the commandlet for
```

Comment:
Does this actually work? I would have thought you would need one of these: ${} $(), not sure which one.
May be worth defining in a variable above.
As a final alternative, I thought MapPaths could cope with fully specified maps (e.g. `/Game/Content/MyMap)`?

Reply:
Issue I was having here was the snapshot was not being found later, as it was looking at the full path (and I guess the snapshot was being dropped into the root of the output directory?). Need a bit more investigation

Reply:
Oh... :D
That's probably a GDK bug, then.